zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -236,6 +237,12 @@ String normalizeNotePath(String notePath) throws IOException {
     }
 
     notePath = notePath.replace("\r", " ").replace("\n", " ");
+
+    notePath = URLDecoder.decode(notePath, "UTF-8");
```

Comment:
It's better to use `URLDecoder.decode(value, StandardCharsets.UTF_8.toString());`
Unfortunately, I have not found a method where you can give a charset.

Reply:
```suggestion
    notePath = URLDecoder.decode(notePath, StandardCharsets.UTF_8.toString());
```

Reply:
CI prints
```Error:  /home/runner/work/zeppelin/zeppelin/zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java:[241,60] package StandardCharsets does not exist```

The StandardCharsets import statement  is missing.
Maybe ```import java.nio.charset.StandardCharsets;```   is ok